recipes/seq-seq-pan/meta.yaml

```diff
@@ -5,8 +5,9 @@ package:
     version: {{ version }}
 
 source:
+    fn: seq-seq-pan.tar.gz
     url: https://gitlab.com/chrjan/seq-seq-pan/repository/v{{ version }}/archive.tar.gz
-    md5: 2135730fee2eec0b018605bef4029f38
+    md5: 7a0c00d8de984f7129c7bcbe4fa26892
     
 build:
     number: 0
```

Comment:
You need to bump the build number if the version didn't change and you want to build a conda package.

Reply:
great, I was just wondering how to get Travis to re-test.